### PR TITLE
added missing commands

### PR DESCRIPTION
**Pull request by benammann**
made a diff between lib/commands.js and `vergd help` and found out lots of commands were missing

camelCase spelling should be correct

will update ruby and php tomorrow

### CODE REVIEW

File: lib/commands.js [hunk 1]
```diff
@@ -55,13 +55,50 @@ var commands = module.exports.commands = [
   , 'walletLock'
   , 'walletPassphrase'
   , 'walletPassphraseChange'
+  ,
+  'checkWallet',
```
Comment (snarkq) [inline lib/commands.js]: Seems odd that formatting is inconsistent with previous entries, could you use a consistent style and restore alphabetic ordering of the list?

Comment (benammann) [reply lib/commands.js]: jeah, that looked kinda ugly

my recent commits fixed that

**Comment by ekryski** [comment]
:shipit:

**Comment by justinvforvendetta** [comment]
@benammann if you can fix the conflict and pull again that would be great. nice changes!

**Comment by benammann** [comment]
@justinvforvendetta thanks, conflict is fixed now

**Comment by benammann** [comment]
going to rework some things, closing this one

new pr in a few hours

**Pull request closed without merging**